packages/typespec-ts/test/modularUnit/scenarios/models/serialization/additionalProperties.md

```diff
@@ -180,79 +180,15 @@ export function complexModelSerializer(item: ComplexModel): any {
     ...serializeRecord(
       item.additionalProperties,
       undefined,
-      simpleModelSerializer
+      simpleModelSerializer,
     ),
-    propA: simpleModelSerializer(item["propA"])
+    propA: simpleModelSerializer(item["propA"]),
   };
 }
 ```
 
 // TODO: enable this case once confirmed https://github.com/microsoft/typespec/issues/6983
 
-# skip: Should generate `additionalProperties` bag for non-legacy code if additional property is overrode
```

Comment:
@v-jiaodi why this is removed?

Reply:
track to issue: https://github.com/Azure/autorest.typescript/issues/3214